Api/Models/Dtos/MenuItem/MenuItemVM.cs

```diff
@@ -30,4 +30,10 @@ public class MenuItemVM
     /// </summary>
     [Range(0, 100)]
     public required decimal? AlcoholPercentage { get; set; }
+
+    /// <summary>
+    /// File name of the photo
+    /// </summary>
+    [Required, StringLength(50)]
+    public required string PhotoFileName { get; set; }
```

Comment:
to nie file name tylko po prostu photo (bo zwracamy ścieżkę do pliku, nie tylko nazwę)